File: Pascal/while-test.pas
program while_test;

var
 a,i: integer;

begin
 a := 1;
 write('How much do you want to count? Insert a number: '); readln(i);
 while a <= i do
 begin
  writeln(a);
  a := a + 1;
 end;

end.

File: C/while_test.c
#include <stdio.h>

int main() {
 int i,a;
 a = 1;

 printf("How much do you want to count? Insert a number: "); scanf("%d", &i);
 
 while(a<=i){
  printf("%d\n", a);
  a++;
 }
 
 return 0;
}
